addons/payment_razorpay/models/payment_provider.py

```diff
@@ -60,7 +76,7 @@ def _get_supported_currencies(self):
             )
         return supported_currencies
 
-    def _razorpay_make_request(self, endpoint, payload=None, method='POST'):
+    def _razorpay_make_request(self, endpoint, payload=None, method='POST', **kwargs):
```

Comment:
```suggestion
    def _razorpay_make_request(self, endpoint, payload=None, method='POST', api_version='v1'):
```

It should be like this as there is not point of variable kwargs